tests/tests.py

```diff
@@ -1612,6 +1612,20 @@ def is_prime(n):
                         i + 1,
                     )
 
+    def test_parser(self):
```

Comment:
Could you rather move this at the end of `test_utils()`, after the Bro parser tests?

Reply:
Done and tests are ok.